api_test.py

```diff
@@ -156,6 +157,34 @@ def test_get_manual_poems(self):
             self.assertEqual(manual_poem['generated'], False)
             self.assertEqual(manual_poem['user_id'], user.id)
 
+    @with_app_context
+    def test_get_some_manual_poems(self):
+        # First create the user to associate poems with
+        user = self.populate_user_data()
+
+        # Create some poems to query for
+        num_poems = 4
+        all_poems = self.populate_poem_data(user.id, False, num_poems)
+        all_poems.sort(
+            key=lambda p: p.modified_timestamp or p.creation_timestamp,
+            reverse=True)
+
+        # Query some of the poems
+        get_partial = self.w.get(
+            f'/api/get_poems/manual/{num_poems//2}/{user.email}')
```

Comment:
super nit: could this be defined as a variable and referenced via the variable here and below?

Reply:
I pulled the url out into a local variable. Or did you mean to turn the `/api/get_poems/manual` part into an instance variable and reference it that way?

Reply:
Oh, I'd meant to comment on just the `num_poems//2` bit since it's referenced throughout the test, it's more meaningful to a reader to see that as a value with a name in multiple places.

Reply:
Ah, ok. I moved that out to a variable in the latest amend.